packages/base/index.d.ts

```diff
@@ -49,12 +51,18 @@ export interface Input {
 export interface Output {
   capacity: HexString;
   lock: Script;
-  type?: Script;
+  type?: Script | null;
```

Comment:
I thin we should either use `?:` syntax, or `| null`, but we should not use both.

Reply:
Is this mean we can't both use `undefined` and `null` to express empty?

If so, I will preserve `null`, and remove `?:`, for `rpc` returns `null`, but it should explicit declaration this key is `null`.

Reply:
I think we should stick to one across the entire lumos codebase, I don't mind whether it is `null` or `undefined`, but I do mind that we use only one.

Reply:
ok

Reply:
`null` removed.